config.json

```diff
@@ -264,144 +264,147 @@
       {
         "groupId": "androidx.compose.animation",
         "artifactId": "animation",
-        "version": "1.5.0",
-        "nugetVersion": "1.5.0",
+        "version": "1.5.1",
+        "nugetVersion": "1.5.1",
         "nugetId": "Xamarin.AndroidX.Compose.Animation",
         "dependencyOnly": false
       },
       {
         "groupId": "androidx.compose.animation",
         "artifactId": "animation-android",
-        "version": "1.5.0",
-        "nugetVersion": "1.5.0",
+        "version": "1.5.1",
+        "nugetVersion": "1.5.1",
         "nugetId": "Xamarin.AndroidX.Compose.Animation.Android",
-        "dependencyOnly": false
+        "dependencyOnly": false,
+        "extraDependencies": "androidx.compose.ui.ui-util-android"
```

Comment:
What is this `extraDependencies` for?

Reply:
1st few binderator runs had

```
snip

43
	System.Exception: 
Artifact
	androidx.compose.ui.ui-unit-android:1.5.0
has unknown 'Runtime' dependency 
	androidx.compose.ui.ui-util:[1.5.0]
Either fulfill or exclude this dependency.

44
	System.Exception: 
Artifact
	androidx.compose.ui.ui-util:1.5.1
has unknown 'Runtime' dependency 
	androidx.compose.ui.ui-util-android:1.5.1
Either fulfill or exclude this dependency.

snip
```

https://github.com/xamarin/AndroidX/pull/783#issuecomment-1713230253

So, I fullfilled it with `extraDependencies `. 

Removed!